test/integration/tests/attestation.sh

```diff
@@ -91,7 +85,7 @@ diff $file_input_data $output_actcredential
 tpm2_pcrreset -Q $debug_pcr
 tpm2_pcrextend -Q $debug_pcr:sha256=$rand_pcr_value
 tpm2_pcrlist -Q
-getrandom 20
+loaded_randomness=$(tpm2_getrandom --hex 20)
```

Comment:
Good catch. I set binary as the default output, but that looks silly. I chose that based on what I though `openssl dgst` was doing, but I was dead wrong. I'm going to change the default to hex and also add a binary option.

Reply:
Wait shouldn't this be a fix to getrandom wrapper?